pkg/pipeline/encode/encode_prom.go

```diff
@@ -348,7 +348,7 @@ func NewEncodeProm(opMetrics *operational.Metrics, params config.StageParam) (En
 	log.Debugf("histos = %v", histos)
 	log.Debugf("aggHistos = %v", aggHistos)
 
-	addr := fmt.Sprintf(":%v", cfg.Port)
+	addr := fmt.Sprintf("%s:%v", cfg.Address, cfg.Port)
```

Comment:
The prom stage is used also by others in existing deployments, so we need to maintain that 0.0.0.0 is the default, even if the user did not specify it.

Reply:
If address is not specified, then we get ":<port>" and not "0.0.0.0:<port>", but that is what we had before, and it works OK.